packages/editor-ui/src/composables/useToast.ts

```diff
@@ -19,17 +19,14 @@ export function useToast() {
 	const externalHooks = useExternalHooks();
 	const i18n = useI18n();
 
-	function showMessage(
-		messageData: Omit<NotificationOptions, 'message'> & { message?: string },
-		track = true,
-	) {
+	function showMessage(messageData: NotificationOptions, track = true) {
 		messageData = { ...messageDefaults, ...messageData };
-		messageData.message = messageData.message
-			? sanitizeHtml(messageData.message)
-			: messageData.message;
+		messageData.message =
+			typeof messageData.message === 'string'
+				? sanitizeHtml(messageData.message)
+				: messageData.message;
```

Comment:
Just curious, what is `message` when not a string?

Reply:
The problem was here we could only pass string and I needed a link (`<a href="">`) and sometimes you need more HTML and the original ElementUI component can also accept `VNODE` or a function returning a `VNODE` so I wanted to make it fully compatible

See `message` in the [docs](https://element-plus.org/en-US/component/message.html#options)